__tests__/app.test.js

```diff
@@ -68,4 +71,23 @@ describe("app.js", () => {
       expect(message).toBe("The article ID must be a number");
     });
   });
+  describe("/api/articles", () => {
+    test("GET 200: responds to the client with an array of articles", async () => {
```

Comment:
I'd update this test description to also say "articles sorted by newest date" as the test also includes the assertion for the sort order